runtime/stdlib/test.go

```diff
@@ -153,7 +176,7 @@ func memberNotFoundError(parentType, member string) error {
 }
 
 var blockchainBackendInterfaceType = func() *sema.InterfaceType {
-	typ, ok := testContractType.NestedTypes.Get(blockchainBackendTypeName)
+	typ, ok := TestContractType().NestedTypes.Get(blockchainBackendTypeName)
```

Comment:
Refactoring `TestContractType` to be initialized lazily is a good start, but the call here and below will cause it to immediately initialized.

To prevent this, the rest of the file must also be refactored in a similar fashion.

Reply:
Ah, I see. My guide here was the comparison with the PR that did the same for the `Crypto` contract. I will update the rest of the values, to be initialized lazily :+1:

Reply:
I found some time to get back to this. It needed quite some rework, to make it truly lazily. But hopefully I got it right this time :pray:

Reply:
The three exported functions that can trigger the initialization of `stdlib` values related to the `Test` contract, are:
- `TestContractChecker()`
- `TestContractType()`
- `TestContractInitializerTypes()` (this is called in `NewTestContract` exported function)